hangups/default.conf

```diff
@@ -0,0 +1,11 @@
+col-scheme: default
+key-prev-tab: ctrl u
+key-next-tab: ctrl d
+key-close-tab: ctrl w
+key-quit: ctrl e
+
+time-format: %I:%M:%S %p
+date-format: %y-%m-%d
+
+#cookies: ~/.cache/hangups/cookies.json
+#log: ~/.cache/hangups/log/hangups.log
```

Comment:
Might be misleading to have these paths here since the default changes based on the platform.